Gems/GraphCanvas/Code/Source/Translation/TranslationBus.h

```diff
@@ -88,10 +88,16 @@ namespace GraphCanvas
         static AZStd::string Sanitize(const AZStd::string& text)
         {
             AZStd::string result = text;
+            AZ::StringFunc::Replace(result, "*", "x");
+            AZ::StringFunc::Replace(result, "(", "_");
+            AZ::StringFunc::Replace(result, ")", "_");
+            AZ::StringFunc::Replace(result, "{", "_");
+            AZ::StringFunc::Replace(result, "}", "_");
             AZ::StringFunc::Replace(result, ":", "_");
             AZ::StringFunc::Replace(result, "<", "_");
             AZ::StringFunc::Replace(result, ",", "_");
             AZ::StringFunc::Replace(result, ">", " ");
+            AZ::StringFunc::Replace(result, "/", "");
```

Comment:
I don't understand the sanitation replacements.
Like "*" being replaced with "x".
What if it contains an actual letter `x`.?

Also right angle bracket being replaced with <space>?

Reply:
It needs to make it compatible with a file format, * is not an allowed character on windows and we have some nodes such as "Multiply (*)". The filename ultimately is not really important, it just has to be valid.